tools/upload_text.php

```diff
@@ -22,7 +22,7 @@
 // days not given (defaults to 0): replace file only
 $days = get_integer_param($_REQUEST, 'days', 0, 0, 56);
 $action = @$_REQUEST['action'];
-$postcomments = @$_POST['postcomments'];
+$postcomments = array_get($_POST, 'postcomments', "");
```

Comment:
Side question: do we have any recommendations around using `array_get` vs the null coalescing operator (`$_POST['postcomment'] ?? null`)?

Reply:
We don't. The rule of thumb I've been using is that if we're getting user attributes (like this) using `array_get()` alongside other `get_*()` functions seems most intuitive. Elsewhere I've used and see both. I don't have strong feelings about this one way or the other.

We've been using `array_get()` before the null coalsecer operator was available to us so we'll have a mix of that for quite a while I expect.

Reply:
Thanks for the context!

As we add more type, I think the null coalescing operator will end up being favored: we can't easily type `array_get` without splitting it by the expected return type, which seems heavy.